### ISSUE TITLE
Web: Add support to copy the text outside of the application

### ISSUE THREAD

**Issue by NathanLovato** [GDQuest/learn-gdscript#1294]
This is a follow-up to issue #1293. Some students would benefit from being able to copy and paste text outside of the app:

- To use in any specific accessibility program or plugin they rely on that's not accessible within the app
- To use with machine translation software when translations are not available in their language or when community translations are incomplete for a given language

Copy and paste is sandboxed in WebGL applications. We have to evaluate if and how this can be achieved reliably today: with Godot's javascript bridge, communicating with the outer page? With a modal popup outside of the application?

Note that this limitation is unique to the web platform; With the desktop releases of the application, people can copy and paste text once #1190 and #1293 are done.
